coq-action.yml.mustache

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    # the OS must be GNU/Linux to be able to use the docker-coq-action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        coq_version:
+{{# tested_coq_opam_versions }}
+          - {{ version }}
+{{/ tested_coq_opam_versions }}
+        ocaml_version: ['4.07-flambda']
```

Comment:
Using `minimal` would sound more robust to me given that some projects may test old Coq versions (which Coq versions support 4.07?).

Reply:
@Zimmi48 good point (the versions of (docker)coq that support 4.07 are documented in the docker-coq wiki, but it looks better to stick to `minimal` indeed)